ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -9,9 +11,14 @@
 REQUEST_CONTEXT_KEY = 'datadog_context'
 REQUEST_SPAN_KEY = '__datadog_request_span'
 
+PARENT_TRACE_HEADER_ID = 'x-ddtrace-parent_trace_id'
+PARENT_SPAN_HEADER_ID = 'x-ddtrace-parent_span_id'
+
+SPAN_MIN_ERROR = 400
```

Comment:
Thanks for this PR! We got many requests for both functionalities (the automatic distributed tracing and choosing the minimum status code that is considered an error). Actually we're still debating internally for both features, because this change is going to impact all other integrations (and even supported languages/frameworks).

Probably I have to keep this PR on-hold for a bit more until we decided how to generally handle that. Anyway thanks for this contribution! We'll give priority to all these specs so that your PR can be merged as soon as possible!